composer.json

```diff
@@ -17,6 +17,7 @@
         "symfony/process": "^4 || ^5 || ^6",
         "symfony/yaml": "^4 || ^5 || ^6",
         "symfony/dotenv": "^5 || ^6",
+        "symfony/filesystem": "^5 || ^6",
```

Comment:
^5 is required for the PHP 7.4 build. Haven't added ^7 as we'd need to update all the other deps and getting this to work for CMS 6 is out of scope